cartofacade/carto_facade_wrapper.go

```diff
@@ -0,0 +1,132 @@
+// Package cartofacade contains the api to call into CGO
```

Comment:
thoughts on file name?

Reply:
Why not just put all these in the existing https://github.com/viamrobotics/viam-cartographer/blob/main/cartofacade/carto_facade.go?

Reply:
I thought it was easier to read to have it separated out, so someone looking at this API wouldn't even need to see the queue implementation. Happy to change it if you think that would be better

Reply:
I think it's confusing to have methods for a type defined in a different file than the type itself.
I think its ok to have both the public api & private methods in the same file as the fact that the private methods are private tell you as the API consumer you don't need to worry about them. I think putting the public methods at the top of the file also help with that.

I vote to put these methods in carto_facade.go

Reply:
done!